### PR TITLE
[DOC] Clarify resources and spans in doc

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:
Adds a sentence clarifying resources and spans to link them to the screenshot in the TraceQL documentation. 

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Comment by 09jvilla** [comment]
Any thoughts on actually putting the added content in the first paragraph? And then maybe moving the screenshot up?

> Attribute fields are derived from the span and can be customized. Process and span attribute types are [defined by the attribute itself](https://github.com/open-telemetry/opentelemetry-proto/blob/b43e9b18b76abf3ee040164b55b9c355217151f3/opentelemetry/proto/common/v1/common.proto#L30-L38), whereas intrinsic fields have a built-in type. You can refer to dynamic attributes (also known as tags) on the span or the span’s resource.

First paragraph seems like its more about the definition of an attribute.

Second paragraph seems a bit more about how attributes are used in the context of TraceQL queries. 

> Attributes in a query start with a span or resource scope depending on what exactly you want to query. This provides significant performance benefits because it allows Tempo to only scan the data you are interested in. To find traces with the GET HTTP method, your query could look like this:

**Comment by 09jvilla** [comment]
> Attributes in a query start with a span or resource scope depending on what exactly you want to query. 

Another idea in case you think "span or resource scope" is not sufficiently clear. 

> Attributes in a query start with a span scope (`span.`) or resource scope (`.resource`) depending on what exactly you want to query.

File: docs/sources/traceql/_index.md [hunk 1]
```diff
@@ -67,11 +67,14 @@ Intrinsic fields are fundamental to spans. These fields can be referenced when s
...
 
-Attributes in a query start with a span or resource scope depending on what exactly you want to query. This provides significant performance benefits because it allows Tempo to only scan the data you are interested in. To find traces with the `GET HTTP` method, your query could look like this:
+Attributes in a query start with a span or resource scope depending on what exactly you want to query. This provides significant performance benefits because it allows Tempo to only scan the data you are interested in.
+
+To find traces with the `GET HTTP` method, your query could look like this:
 
 ```
 { span.http.method = "GET" }
 ```
+There are two types of attributes, span attributes and resource attributes. By expanding a span in the Grafana UI, you can see both its span attributes (1 in the screenshot) and resource attributes (2 in the screenshot).
```
Comment (knylander-grafana) [inline docs/sources/traceql/_index.md:77]:
```suggestion

There are two types of attributes: span attributes and resource attributes. By expanding a span in the Grafana UI, you can see both its span attributes (1 in the screenshot) and resource attributes (2 in the screenshot).
```

File: docs/sources/traceql/_index.md [hunk 2]
```diff
@@ -67,12 +67,16 @@ Intrinsic fields are fundamental to spans. These fields can be referenced when s
...
+
+To find traces with the `GET HTTP` method, your query could look like this:
 
 ```
 { span.http.method = "GET" }
 ```
 
+There are two types of attributes: span attributes and resource attributes. By expanding a span in the Grafana UI, you can see both its span attributes (1 in the screenshot) and resource attributes (2 in the screenshot).
+
 <p align="center"><img src="assets/span-resource-attributes.png" alt="Example of span resources and attributes" /></p>
```
Comment (knylander-grafana) [inline docs/sources/traceql/_index.md:80]:
```suggestion
<p align="center"><img src="assets/span-resource-attributes.png" alt="Example of span and resource  attributes." /></p>
```

File: docs/sources/traceql/_index.md [hunk 3]
```diff
@@ -67,13 +67,17 @@ Intrinsic fields are fundamental to spans. These fields can be referenced when s
...
+To find traces with the `GET HTTP` method, your query could look like this:
 
 ```
 { span.http.method = "GET" }
 ```
 
-<p align="center"><img src="assets/span-resource-attributes.png" alt="Example of span resources and attributes" /></p>
+There are two types of attributes: span attributes and resource attributes. By expanding a span in the Grafana UI, you can see both its span attributes (1 in the screenshot) and resource attributes (2 in the screenshot).
+
+<p align="center"><img src="assets/span-resource-attributes.png" alt="Example of span and resource  attributes." /></p>
```
Comment (knylander-grafana) [inline docs/sources/traceql/_index.md:81]:
```suggestion

For more information about attributes and resources, refer to the [OpenTelemetry Resource SDK](https://opentelemetry.io/docs/reference/specification/resource/sdk/).
```

**Review by joe-elliott — APPROVED** [review]

**Comment by knylander-grafana** [comment]
@09jvilla Thank you for the comments. I rearranged the sentences and screenshot per your suggestion. I also added an earlier inline example to help clarify.

**Pull request merged**